mesa/Person.py

```diff
@@ -91,33 +96,68 @@ def step(self):
             pass
 
     def randomfriends(self):
+        """
```

Comment:
Better changing the name if we're touching here, it doesn't reflect what the function does.